pkg/forwarder/forwarder.go

```diff
@@ -551,8 +552,15 @@ func (f *DefaultForwarder) SubmitV1CheckRuns(payload Payloads, extra http.Header
 }
 
 // SubmitV1Intake will send payloads to the universal `/intake/` endpoint used by Agent v.5
-func (f *DefaultForwarder) SubmitV1Intake(payload Payloads, extra http.Header, priority TransactionPriority) error {
-	transactions := f.createPriorityHTTPTransactions(v1IntakeEndpoint, payload, true, extra, priority)
+func (f *DefaultForwarder) SubmitV1Intake(payload Payloads, extra http.Header) error {
+	return f.submitV1Intake(payload, extra, f.createHTTPTransactions)
+}
+
+func (f *DefaultForwarder) submitV1Intake(
```

Comment:
Could we rename this function, having 2 functions with the same name in upper and lower case is confusing.